### DISCUSSION TITLE
Erro no SEI Pro

### DISCUSSION THREAD

**Discussion by pedroticpi** [community/community#49709] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bom dia.
Estou com esse erro na Extensão para Chrome
![image](https://user-images.githubusercontent.com/65689239/224344071-f03b961c-605d-4aba-9cc1-db5807bff506.png)

**Comment by otter-computer** [comment]
Hi @pedroticpi, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
